hoomd/mpcd/CollisionMethod.cu

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2009-2025 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+/*!
+ * \file mpcd/CollisionMethod.cu
+ * \brief Defines GPU functions and kernels used by mpcd::CollisionMethod
+ */
+
+#include "CollisionMethod.cuh"
+
+namespace hoomd
+    {
+namespace mpcd
+    {
+namespace kernel
+    {
```

Comment:
You'll need to add the extra GPU namespace. You've then put the kernels in the right spot, but then you also need to have the driver functions.
```suggestion
namespace gpu
    {
namespace kernel
    {
```